tsl/src/hypercore/utils.c

```diff
@@ -12,13 +12,33 @@
 #include <catalog/pg_class.h>
 #include <commands/defrem.h>
 #include <nodes/makefuncs.h>
+#include <storage/lockdefs.h>
 #include <utils/builtins.h>
 #include <utils/lsyscache.h>
 #include <utils/syscache.h>
 
+#include "chunk.h"
 #include "extension_constants.h"
+#include "src/utils.h"
 #include "utils.h"
-#include <src/utils.h>
+
+void
+hypercore_set_reloptions(Chunk *chunk)
```

Comment:
Suggest adding function-level comment on what this does (which options are set) and why.